src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -281,6 +281,52 @@ Below is a list of available validate commands:
 | `--kubeconfig` `<string>`    | Absolute path(s) to the kubeconfig file(s) (default `$HOME/.kube/config`)
 | `--kubecontext` `<string>`   | Kubeconfig context to use
 
+### `gather`
+
+The `subctl gather` command is tool that collects various information from clusters to aid in troubleshooting a
+Submariner deployment, including Kubernetes resources and Pod logs. Clusters from which information is gathered are provided
+through `--kubeconfig` flag (or `KUBECONFIG` env var). By default it will gather information from all clusters contexts provided
+in kubeconfig. To gather information from sepecific clusters, contexts can be passed using `kubecontexts` flag.
```

Comment:
wasn't there a comment somewhere from @skitt to keep this just `kubecontext`?

Reply:
Maybe for other commands and backward compatibility. But this is a new command and we want to use kubecontexts.